smarts/waymo/waymo_utils.py

```diff
@@ -24,10 +24,14 @@
 from matplotlib import pyplot as plt
 from matplotlib.animation import FuncAnimation
 from matplotlib.lines import Line2D
-from waymo_open_dataset.protos import scenario_pb2
-
 from smarts.core.utils.file import read_tfrecord_file
-from smarts.core.waymo_map import WaymoMap
```

Comment:
Right, this removal should help with the bleeding import errors related to waymo.